src/transform/dive_effort_to_twiststamped.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import rospy
```

Comment:
uuv simulator doesn't use stamped messages anywhere. It will be easier to make changes in our code than uuv. Also, I think we don't really stamped messages.

Reply:
I would stay with stamped messages - timestamps are useful!